src/main/java/com/to/ordermanagementservice/BasketTest.java

```diff
@@ -0,0 +1,36 @@
+package com.to.ordermanagementservice;
+
+import com.to.ordermanagementservice.entity.*;
+import com.to.ordermanagementservice.enums.Genre;
+import com.to.ordermanagementservice.repository.BasketRepository;
+import com.to.ordermanagementservice.repository.impl.BasketRepositoryImpl;
+
+import java.math.BigDecimal;
+import java.time.OffsetDateTime;
+
+public class BasketTest {
+    public static void main(String[] args) {
+        BasketRepository basket = new BasketRepositoryImpl();
+        Electronic electronic1 = new Electronic(1, "TV",new BigDecimal(50000),"SAMSUNG",OffsetDateTime.now());
+        basket.addProduct(electronic1);
+        Electronic electronic2 = new Electronic(2, "PS",new BigDecimal(25000),"SONY",OffsetDateTime.now());
+        basket.addProduct(electronic2);
+        Electronic electronic3 = new Electronic(3, "Phone",new BigDecimal(300),"Nokia",OffsetDateTime.now());
+        basket.addProduct(electronic3);
+        Grocery grocery1 = new Grocery(4,"bread",new BigDecimal(23),OffsetDateTime.now());
+        basket.addProduct(grocery1);
+        Grocery grocery2 = new Grocery(5,"milk",new BigDecimal(42),OffsetDateTime.now());
+        basket.addProduct(grocery2);
+        Grocery grocery3 = new Grocery(6,"meat",new BigDecimal(150),OffsetDateTime.now());
+        basket.addProduct(grocery3);
+        Clothing clothing1 = new Clothing(7,"ClubTS",new BigDecimal(20),50,"Blue");
+        basket.addProduct(clothing1);
+        Clothing clothing2 = new Clothing(8,"Pants",new BigDecimal(40),48,"yellow");
+        basket.addProduct(clothing2);
+        Book book1 =new Book(9,"Divine comedy",new BigDecimal(1.80),"Dante Alegrei",1680,238,Genre.MYSTERY);
+        basket.addProduct(book1);
+        Book book2 =new Book(10,"Kobzar",new BigDecimal(2.80),"Tarass Shevchenko",1820,1000000,Genre.ROMANCE);
+        basket.addProduct(book2);
```

Comment:
Check Java code formatting.

You can also add empty line each time when you add a specific product to a basket so that operations for each product object were separated and code becomes more readable

Reply:
add empty lien